content/en/docs/pipelines/sdk/python-function-components.ipynb

```diff
@@ -53,14 +54,19 @@
    "outputs": [],
    "source": [
     "import kfp\n",
-    "import kfp.components as comp"
+    "from kfp.components import create_run_from_pipeline_func, InputPath, OutputPath"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "3. Create an instance of the [`kfp.Client` class][kfp-client].\n",
+    "3. Create an instance of the [`kfp.Client` class][kfp-client]. To find your\n",
+    "   Kubeflow Pipelines cluster's host name, open the Kubeflow Pipelines user\n",
+    "   interface in your browser. The URL of the Kubeflow Pipelines user\n",
+    "   interface is something like \n",
+    "   `https://my-cluster.my-organization.com/#/pipelines`. In this case, the\n",
```

Comment:
There are other forms of endpoint and host at least. With Kubeflow deployment, the endpoint for Kubeflow will be in the format of
`https://<KF_NAME>.endpoints.<PROJECT>.cloud.goog`
and the host value for kfp client needs to be 
`https://<KF_NAME>.endpoints.<PROJECT>.cloud.goog/pipeline`

Reply:
I think that we should use a general version of KFP URL, rather than a cloud specific version. I've updated to use the "/pipeline" format of the URL without a cloud provider specific domain.